indico/modules/rb_new/schemas.py

```diff
@@ -77,6 +77,13 @@ class Meta:
         fields = ('id', 'booking_reason', 'booked_for_name')
 
 
+class ReservationEventDataSchema(mm.ModelSchema):
```

Comment:
don't use `ModelSchema` if you don't base it on an actual modal class (specified in the `class Meta`); instead use a normal `mm.Schema`.. Or you could use a ModelSchema with `model = Event` and simply restrict the fields. That way you don't have to define the types of the fields since they'll be taken from the Event model.

Reply:
Yeah, copy-paste.